sandboxes/crates.io/Dockerfile

```diff
@@ -5,14 +5,18 @@ RUN apt-get update && \
         curl \
         wget \
         git \
-        python
+        python \
+		sudo
```

Comment:
nit: the indent here (and the other files) doesn't look correct.

Reply:
Whoops, it looked fine in my editor but it was actually putting in tabs instead of spaces